index.html

```diff
@@ -1832,55 +1832,141 @@ <h2>Proof</h2>
-JSON-LD syntax, a format for mapping JSON data into the RDF semantic graph
-model, as defined by [[!JSON-LD]]. This section formalizes how the data model
-(described in Sections <a href="#data-model"></a> and
-<a href="#did-documents"></a>) is realized in JSON-LD.
+All concrete representations of a <a>DID document</a> MUST be serialized using a 
+deterministic mapping that is able to be unambiguously parsed into the data model
+defined in this specification. All serialization methods MUST define rules for 
+the bidirectional translation of a <a>DID document</a> both into and out of the
+representation in question. As a consequence, translation between any two representations
+MUST be done by parsing the source format into a <a>DID document</a> model
+(described in Sections <a href="#data-model"></a> and <a href="#did-documents"></a>)
+and then serializing the <a>DID document</a> model into the target representation. An
+implementation MUST NOT convert between representations without first parsing to a 
+<a>DID document</a> model. 
       </p>
+
+      <p>
+Although syntactic mappings are provided for JSON, JSON-LD, and CBOR here, applications
+and services MAY use any other data representation syntax that is capable of expressing 
+the data model, such as XML or YAML.
+      </p>
+      
+      <p>
+Producers MUST indicate which representation of a document has been used, and consumers
+MUST be made aware of which representation a document is in.
+      </p>
+      
       <p>
-Although syntactic mappings are provided for JSON and JSON-LD only, applications
-and services can use any other data representation syntax, such as JXD (JSON XDI
-Data, a serialization format for the
-<a href="http://docs.oasis-open.org/xdi/xdi-core/v1.0/csd01/xdi-core-v1.0-csd01.xml">XDI graph model</a>),
-XML, YAML, or CBOR, that is capable of expressing the data model.
+The production and consumption rules in this section apply to all implementations
+seeking to be fully compatible with independent implementations of the specification.
+Deployments of this specification MAY use a custom agreed-upon representation, including
+localized rules for handling properties not listed in the registry.
+See section ?? on Extensibility for more information.
       </p>
 
     <section>
       <h2>
-Pure JSON
+JSON
       </h2>
+      
+      <p>
+When producing and consuming DID documents that are in plain JSON (as noted by
+the document metadata), the following rules MUST be followed. 
+      </p>
+
+<section>
+      <h3>Production</h3>
 
         <p>
-The data model, as described in Section <a href="#data-model"></a>, can be
-encoded in Javascript Object Notation (JSON) [[RFC8259]] by mapping property
+A <a>DID document</a> MUST be a single JSON object conforming to [[!RFC8259]]. All
+top-level properties of the <a>DID document</a> MUST be represented by using the
+property name as the name of the member of the JSON object. The values of
```

Comment:
```suggestion
property as the name of the member of the JSON object. The values of
```

Reply:
I'm ok with this change, but we'll need to clarify that by "property" here we mean the string for the property itself and not any other piece. I'll add an issue (and link) for that here.